src/components/AboutUs/Partners.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import { Padding } from 'styled-components-spacing'
+
+import Image from '../Common/Image'
+import { Grid, Col, Row } from '../grid'
+import { SectionTitle } from '../Typography'
+
+const PaddedCol = styled(Col)`
+  flex-direction: column;
+  align-items: flex-start;
+
+  ${breakpoint('smallPhone')`
+    padding-bottom: ${props => props.theme.spacing[1]}
+    &:last-child {
+      padding-bottom: 0
+    }
+  `}
+  ${breakpoint('tablet')`
+    padding-bottom: 0
+  `}
+`
+
+const Partners = ({ partnershipsTitle, partners }) => (
+  <Grid>
+    <Padding
+      top={{ smallPhone: 3, tablet: 4 }}
+      bottom={{ smallPhone: 3.5, tablet: 5 }}
+    >
+      <Padding bottom={{ smallPhone: 3, tablet: 4 }}>
+        <SectionTitle>{partnershipsTitle}</SectionTitle>
+      </Padding>
+      <Row>
+        {partners.map((partner, idx) => (
+          <PaddedCol
```

Comment:
minor change, but I'd suggest doing the same as in the `Subsidiaries` — creating a `Partner` Component that does what this block does and use it here : )

Reply:
I get what you mean, but In the subsidiaries PR, the `PaddedCol` is not part of `Subsidiary`. If I follow the same pattern here, then `Partner` = `Image`

Reply:
Ok, I've just moved this into a separate sub-component called `Partner`

I want to copy this on Subsidiaries too - put `PaddedCol` inside Subsidiary, but for some reason that I cannot fathom, it breaks the layout :confused: 
![image](https://user-images.githubusercontent.com/15656538/53088065-948c3700-3500-11e9-9c42-f273c9e7d339.png)

Reply:
Ok, I see. It's this bit inside `ColumnLayout`
![image](https://user-images.githubusercontent.com/15656538/53088355-5f341900-3501-11e9-9937-edcfa81b20f4.png)

Probably, it's not worth using this component. I'm gonna replace it with an explicit `Row` + `Col`, like in Partners